src/payment/PaymentPage.jsx

```diff
@@ -27,11 +27,15 @@ class PaymentPage extends React.Component {
     const {
       experimentVariables: {
         isPaymentVisualExperiment = false,
+        isNumEnrolledExperiment = false,
+        enrollmentCountData = [],
```

Comment:
Are you adding this data via optimizely? I guess that's fine if Shelby said we don't want to put the full json in the code

Reply:
I am adding via Optimizely to the window Object. Matthew suggested to add this as a JSON file in PaymentMFE since I am making changes might as well add it to the code